doc/api/buffer.md

```diff
@@ -1875,10 +1892,10 @@ console.log(`${len} bytes: ${buf.toString('utf8', 0, len)}`);
 ### buf.writeDoubleBE(value, offset[, noAssert])
 ### buf.writeDoubleLE(value, offset[, noAssert])
 
-* `value` {Number} Bytes to be written to Buffer
-* `offset` {Number} `0 <= offset <= buf.length - 8`
-* `noAssert` {Boolean} Default: false
-* Return: {Number} The offset plus the number of written bytes
+* `value` {Number} Number to be written to `buf`
+* `offset` {Number} Where to start writing. Must satisfy: `0 <= offset <= buf.length - 8`
+* `noAssert` {Boolean} Skip `value` and `offset` validation? **Default:** `false`
```

Comment:
Maybe we should make explicit that the value written to an out of range number with `noAssert` is UB. Because right now it's very predictable, and easy to predict. For example:
```js
const b = Buffer.alloc(4);
b.writeInt32LE((-1 >>> 0) - 5, 0, true);
b.readUInt32LE(0) === (-1 >>> 0) - 5;
```